web/app/view/dialog/User.js

```diff
@@ -175,6 +175,31 @@ Ext.define('Traccar.view.dialog.User', {
                     }
                 }
             }]
+        }, {
+            xtype: 'fieldset',
```

Comment:
I can move it next to token but added a new fieldset for 2FA authentication methods. It could be that in the future traccar also support another 2FA method like sms text or ...
Let me know what you prefer. As previously commented the totp key is generated server side an the user has no influence over than that it can request a new totp key.